manifests/init.pp

```diff
@@ -148,5 +153,16 @@
 
   Service['httpd'] -> Exec['foreman-rake-db:seed']
 
+  file { $repo_export_dir:
```

Comment:
This seems more appropriate in the config manifest based on our typical separation of duties